functions-and-scope.js

```diff
@@ -16,6 +16,15 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+console.log("Opdracht 1a");
+
+let counter = 0;
+for (let i = 0; i < grades.length; i++) {
+    if (grades[i] >= 8) {
+        counter += 1;
```

Comment:
Je zou dit nog iets kunnen versimpelen:
```suggestion
        counter++;
```